lib/bootloader_setup.pm

```diff
@@ -280,8 +280,8 @@ sub bootmenu_network_source {
             my ($m_server, $m_share, $m_directory);
 
             # Parse SUSEMIRROR into variables
-            if ($m_mirror =~ m{^[a-z]+://([a-zA-Z0-9.-]*)(/.*)$}) {
-                ($m_server, $m_directory) = ($1, $2);
+            if ($m_mirror =~ m{^[a-z]+://(?<server>[a-zA-Z0-9.-]*)/(?<dir>.*)$}) {
+                ($m_server, $m_directory) = ($+{server}, $+{dir});
                 if ($m_protocol eq "smb") {
                     ($m_share, $m_directory) = $m_directory =~ /\/(.+?)(\/.*)/;
```

Comment:
Are you not breaking smb installations here?

Reply:
You're right, it would break smb and nfs installations. Problem is only for ftp, which actually works fine, but it's not failing test. Nice catch, thanks!